src/quemb/molbe/autofrag.py

```diff
@@ -332,6 +417,12 @@ def graphgen(
     for adx, _ in enumerate(fragment_map.fs):
         fragment_map.dnames.append(str(frag_prefix) + str(adx))
 
+    if export_graph_to:
+        fragment_map.export_graph(
+            outdir=str(export_graph_to),
```

Comment:
Any reason to cast PathLike to str when the function accepts PathLike?

Reply:
To second @mscho527 . It is actually dangerous to cast to string here and I would just pass it on